src/features/navigation/Header.jsx

```diff
@@ -27,8 +28,11 @@ import {
   selectConnected,
   selectNiceName,
   selectTemperature,
+  selectHasAdb,
```

Comment:
Please sort alphabetically - I know it's not consistent everywhere - I would still prefer it to be ;-)